src/xform_to_datalog/datalog_facts_test.cc

```diff
@@ -55,13 +56,17 @@ INSTANTIATE_TEST_SUITE_P(
 .decl duplicateTestCaseNames(testAspectName: symbol)
 .output duplicateTestCaseNames(IO=stdout)
 
+.output disallowedUsage(IO=stdout)
```

Comment:
Why is the `.output disallowedUsage(IO=stdout)` declaration in two locations?

Reply:
You mean a few different files? Two of these files (like the one in this comment) contain a string that is used to compare against the output of generated code. So we need to change both the code generation in `datalog_facts.h` and the code used for tests (twice in `datalog_facts_test.cc`, `ok_claim_propagates.dl`). Cloudbuild is sad if we don't do this.